content/Foundation/Membership/LOGOS/secondstate.svg

```diff
@@ -1,71 +1,132 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 595.28 595.28" style="enable-background:new 0 0 595.28 595.28;" xml:space="preserve">
-<style type="text/css">
+
+<svg
+   version="1.1"
+   x="0px"
+   y="0px"
+   viewBox="0 130 595.28 335"
+   style="enable-background:new 0 0 595.28 595.28;"
+   xml:space="preserve"
+   sodipodi:docname="logo.svg"
+   inkscape:version="1.1 (c4e8f9e, 2021-05-24)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg"><defs
+   id="defs47"><clipPath
+     clipPathUnits="userSpaceOnUse"
+     id="clipPath153"><rect
+       style="fill:#0000ff;fill-rule:evenodd;stroke-width:1.73553"
+       id="rect155"
+       width="610"
+       height="334.89999"
+       x="-10.365088"
+       y="130.19099" /></clipPath></defs><sodipodi:namedview
+   id="namedview45"
+   pagecolor="#ffffff"
+   bordercolor="#666666"
+   borderopacity="1.0"
+   inkscape:pageshadow="2"
+   inkscape:pageopacity="0.0"
+   inkscape:pagecheckerboard="0"
+   showgrid="false"
+   inkscape:zoom="1.4665367"
+   inkscape:cx="297.29908"
+   inkscape:cy="297.64001"
+   inkscape:window-width="1803"
+   inkscape:window-height="1081"
+   inkscape:window-x="0"
+   inkscape:window-y="0"
+   inkscape:window-maximized="0"
+   inkscape:current-layer="图层_1" />
```

Comment:
drop the chinese chars here?

Reply:
I don't think they are a problem, svg/xml is perfectly fine with utf8 and it does specify utf8 as the encoding here. Chinese characters are no more special than any other ones.